README.md

```diff
@@ -10,13 +10,23 @@ components. It is built on top of [reactstrap][reactstrap-homepage].
 
 ## Installation
 
-*dash-bootstrap-components* is hosted on PyPI, and can be installed by
-running
+### PyPI
+
+You can install *dash-bootstrap-components* with `pip`:
 
 ```
 pip install dash-bootstrap-components
 ```
 
+### Anaconda
+
+Anaconda users can install *dash-bootstrap-components* through the
```

Comment:
(Nit) Anaconda is the distribution, and conda the package manager. You can use conda, and install dash-bootstrap-components with conda, without using Anaconda.

Reply:
How about:

```
### Anaconda

You can also install *dash-bootstrap-components* with `conda` through the conda-forge channel:
```

Reply:
That sounds fine to me!